docker-compose.katana.yaml

```diff
@@ -0,0 +1,56 @@
+version: "3.2"
+
+services:
+  katana:
+    image: ghcr.io/dojoengine/dojo:latest
+    command: 
+      - "katana"
+      - "--allow-zero-max-fee"
+      - "--validate-max-steps"
+      - "16777216"
+      - "--invoke-max-steps"
+      - "16777216"
+      - "--gas-price"
+      - "0"
+    ports:
+      - 5050:5050
+    restart: on-failure
+    networks:
+      - internal
+
+  kakarot-deployer:
+    image: ghcr.io/kkrt-labs/kakarot/deployer:latest
+    environment:
+      - STARKNET_NETWORK=katana
+      - KATANA_PRIVATE_KEY=${KATANA_PRIVATE_KEY}
+      - KATANA_ACCOUNT_ADDRESS=${KATANA_ACCOUNT_ADDRESS}
+      - KATANA_RPC_URL=http://katana:5050
+    volumes:
+      - ./deployments:/app/kakarot/deployments
+    depends_on:
+      - katana
+    networks:
+      - internal
+
+  kakarot-rpc:
+    image: ghcr.io/kkrt-labs/kakarot-rpc/node:latest
+    ports:
+      - 3030:3030
+    entrypoint: ["./run.sh", "/usr/bin/tini", "--", "/usr/local/bin/kakarot-rpc"]
+    environment:
+      - KAKAROT_HTTP_RPC_ADDRESS=0.0.0.0:3030
+      - STARKNET_NETWORK=katana
+      - KATANA_RPC_URL=http://katana:5050
+      - RUST_LOG=trace
```

Comment:
we need the deployer process to fund an eoa and usually set the private key to match with the first test account of anvil, so setting
`EVM_PRIVATE_KEY=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80` right here would help make the forge script easily runnable!

Reply:
Made `EVM_PRIVATE_KEY` configurable and the one from `.env`